docs/gettingstarted/broadcast_data.md

```diff
@@ -67,7 +67,7 @@ Status: `202 Accepted` - the message is on it's way, but has not yet been confir
     "author": "0x0a65365587a65ce44938eab5a765fe8bc6532bdf", // set automatically in this example to the node org
```

Comment:
@nguyer - have we moved this guidance documentation on the example result to somewhere else?

Reply:
ok - understand now, this isn't Git saying there's a diff, it's just pointing out that we've got non-JSON in a JSON "```json" block (which we did intentionally as it highlights better in our docs theme than "```javascript`